etc/kayobe/kolla/globals.yml

```diff
@@ -9,62 +9,15 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts
 # kolla_base_distro must be set here to be resolvable on a per-host basis
 # This is necessary for os migrations where mixed clouds might be deployed
 kolla_base_distro: "{% raw %}{{ 'centos' if ansible_facts.distribution == 'Rocky' and ansible_facts.distribution_major_version == '8' else ansible_facts.distribution | lower }}{% endraw %}"
-magnum_tag: yoga-20231002T101716
 opensearch_tag: yoga-20230904T121340
-octavia_tag: yoga-20230921T074238
 
+# Live-migration is broken from qemu-kvm-6.2.0-20.module_el8.7.0+1218+f626c2ff to qemu-kvm-6.2.0-39.module_el8+669+76cc32af
+# with signature: `Missing section footer for 0000:00:01.3/piix4_pm`. Test nova carefully before bumping.
```

Comment:
If this comment still applies then we still need the old nova_libvirt tag. If not then the comment can be removed.

Reply:
Fixes PR: https://github.com/stackhpc/stackhpc-kayobe-config/pull/723